/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/cli.py:617: UserWarning: `System.configure_optimizers` will be overridden by `LightningCLI.configure_optimizers`.
  _warn(
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
[rank: 0] Global seed set to 42
Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/2
----------------------------------------------------------------------------------------------------
distributed_backend=nccl
All distributed processes registered. Starting with 2 processes
----------------------------------------------------------------------------------------------------
0it [00:00, ?it/s]
0it [00:00, ?it/s]
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1,2,3]
  | Name                 | Type                    | Params
-----------------------------------------------------------------
0 | model                | MixStyleTransferModel   | 26.0 M
1 | generate_mix_console | AdvancedMixConsole      | 0
2 | loss                 | MultiResolutionSTFTLoss | 0
3 | sisdr                | SISDRLoss               | 0
4 | mrstft               | MultiResolutionSTFTLoss | 0
-----------------------------------------------------------------
26.0 M    Trainable params
0         Non-trainable params
26.0 M    Total params
104.185   Total estimated model params size (MB)
Traceback (most recent call last):
  File "/homes/ssv02/Diff-MST/main.py", line 26, in <module>
    cli_main()
  File "/homes/ssv02/Diff-MST/main.py", line 15, in cli_main
    cli = LightningCLI(
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/cli.py", line 353, in __init__
    self._run_subcommand(self.subcommand)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/cli.py", line 642, in _run_subcommand
    fn(**fn_kwargs)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 520, in fit
    call._call_and_handle_interrupt(
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/trainer/call.py", line 42, in _call_and_handle_interrupt
    return trainer.strategy.launcher.launch(trainer_fn, *args, trainer=trainer, **kwargs)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/strategies/launchers/subprocess_script.py", line 92, in launch
    return function(*args, **kwargs)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 559, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 935, in _run
    results = self._run_stage()
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 976, in _run_stage
    self._run_sanity_check()
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py", line 1005, in _run_sanity_check
    val_loop.run()
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/loops/utilities.py", line 177, in _decorator
    return loop_run(self, *args, **kwargs)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/loops/evaluation_loop.py", line 108, in run
    batch, batch_idx, dataloader_idx = next(data_fetcher)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/loops/fetchers.py", line 136, in __next__
    self._fetch_next_batch(self.dataloader_iter)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/loops/fetchers.py", line 150, in _fetch_next_batch
    batch = next(iterator)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/utilities/combined_loader.py", line 276, in __next__
    out = next(self._iterator)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/pytorch_lightning/utilities/combined_loader.py", line 122, in __next__
    out = next(self.iterators[self._iterator_idx])
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 634, in __next__
    data = self._next_data()
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1346, in _next_data
    return self._process_data(data)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1372, in _process_data
    data.reraise()
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/_utils.py", line 644, in reraise
    raise exception
ValueError: Caught ValueError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/utils/data/_utils/worker.py", line 308, in _worker_loop
    data = fetcher.fetch(index)
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/homes/ssv02/Diff-MST/env/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/homes/ssv02/Diff-MST/mst/dataloaders/medley.py", line 268, in __getitem__
    example_idx = np.random.randint(0, len(self.examples))
  File "mtrand.pyx", line 748, in numpy.random.mtrand.RandomState.randint
  File "_bounded_integers.pyx", line 1247, in numpy.random._bounded_integers._rand_int64
ValueError: high <= 0
Found 0 mix directories in ['/scratch/medleydb/V1', '/scratch/medleydb/MedleyDB_V2'].
Found 0 mix directories in train set.
Found 0 mix directories with tracks less than 20 and more than 4.
Found 0 mix directories in ['/scratch/medleydb/V1', '/scratch/medleydb/MedleyDB_V2'].
Found 0 mix directories in val set.
Found 0 mix directories with tracks less than 20 and more than 4.
Sanity Checking: 0it [00:00, ?it/s]